src/plugins/mf_priority.cpp

```diff
@@ -201,8 +201,8 @@ static int conf_update_cb (flux_plugin_t *p,
     }
 
     // assign unpacked weights into priority_weights map
-    priority_weights["fairshare"] = fshare_weight;
-    priority_weights["queue"] = queue_weight;
+    if (fshare_weight != -1) priority_weights["fairshare"] = fshare_weight;
+    if (queue_weight != -1) priority_weights["queue"] = queue_weight;
```

Comment:
I find single line conditionals a little hard to read, I'd suggest breaking these into two lines for readability.
I see we didn't comment on this earlier so this is just a suggestion not required.

Reply:
No problem - I think that's perfectly reasonable. I'll push up a fix to break these up into two lines. 👍